Makefile

```diff
@@ -47,6 +49,24 @@ build: bin $(CARGO_TARGET_DIR)
 	cargo build $(release)
 	cp $(CARGO_TARGET_DIR)/$(profile)/netavark bin/netavark$(if $(debug),.debug,)
 
+.PHONY: crate-publish
+crate-publish:
+	@if [ "$(publish-version)" = "" ]; then\
+		echo "publish-version not set. Usage example 'make crate-publish publish-version=v0.1.2'";\
+		echo "Make sure repo already contains tag for the version";\
+		exit 1;\
+	fi
+	@if [ "$(REMOTE_ORIGIN_URL)" != "https://github.com/containers/netavark" ]; then\
+		echo "You are working with a fork please use actual repo while publishing. Please clone https://github.com/containers/netavark";\
+		echo "Make sure repo already contains tag for $(publish-version)";\
+		exit 1;\
+	fi
```

Comment:
This looks wrong. Why am I forced to clone upstream as origin? My fork already has all the correct tags in it.

Reply:
`cargo` does treat publishing from another repo as a clean publish, you will have to force `--allow-dirty` which will still work but its not the correct way.

Reply:
@Luap99 let me try again if `cargo` warns when publishing from a fork or not.

Reply:
removed the check i think fork is also allowed